tests/serverpod_test_server/test_integration/auth_core/auth_header_test.dart

```diff
@@ -0,0 +1,161 @@
+import 'dart:async';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/test_key_manager.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  late Completer<String> tokenInspectionCompleter;
+  Future<AuthenticationInfo?> authenticationHandler(
+    Session session,
+    String token,
+  ) {
+    tokenInspectionCompleter.complete(token);
+    return Future.value(AuthenticationInfo(1, {}));
+  }
+
+  group('When passing auth key in HTTP header format', () {
```

Comment:
Suggestion: These tests are missing their `Given` statement.

For example `Given auth key in HTTP header format When calling endpoint then endpoint method should receive plain auth key (i.e. in original format)`.